### PR TITLE
menu-order code push 

### PR DESCRIPTION
**Pull request by vc-bhanudatsinh**
I have created the CRUD APIs for pizza, burger and pasta foods.

### CODE REVIEW

File: menu-order/db/db.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "pizza": [
+    { "itemId": 1234, "name": "cheezy7", "prize": 20 },
```
Comment (vc-mohit) [inline menu-order/db/db.json]: key itself is not correct @vc-bhanudatsinh

File: menu-order/src/routes/pastaRoutes.js
Comment (vc-mohit) [inline menu-order/src/routes/pastaRoutes.js:1]: All path names should be in camelCase

File: menu-order/src/routes/pizzaRoutes.js
Comment (vc-mohit) [inline menu-order/src/routes/pizzaRoutes.js:1]: All path names should be in camelCase

File: menu-order/src/routes/pastaRoutes.js
Comment (vc-bhanudatsinh) [reply menu-order/src/routes/pastaRoutes.js:1]: Please check I have used camelCase format only. except the public folder

**Review by vc-mohit — DISMISSED** [review]

File: menu-order/src/routes/pastaRoutes.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as express from "express";
+
+import * as pastaController from "../controllers/pastaController.js";
+
+const pastaRouter = express.Router();
+
+pastaRouter.get("/getitem", pastaController.getPasta);
```
Comment (vc-mohit) [inline menu-order/src/routes/pastaRoutes.js]: All path names should be in camelCase.
getitem -> getItem

File: menu-order/src/routes/pizzaRoutes.js
Comment (vc-bhanudatsinh) [reply menu-order/src/routes/pizzaRoutes.js:1]: done

File: menu-order/src/routes/pastaRoutes.js
Comment (vc-bhanudatsinh) [reply menu-order/src/routes/pastaRoutes.js:1]: done

**Review by vc-mohit — APPROVED** [review]

**Comment by vc-bhanudatsinh** [comment]
Thank you so much @vc-mohit bhai. bhagwan aapka bhala kare

**Comment by gaurangviitorcloud** [comment]
> 

Please use this space for conversations related to PR only and keep it professional

**Comment by vc-mohit** [comment]
@vc-bhanudatsinh: Don't send this type of message, Behave like a Professional.

**Pull request merged**